src/mtype.cpp

```diff
@@ -200,19 +222,9 @@ itype_id mtype::get_meat_itype() const
 
 int mtype::get_meat_chunks_count() const
 {
-    switch( size ) {
-        case MS_TINY:
-            return 2;
-        case MS_SMALL:
-            return 64;
-        case MS_MEDIUM:
-            return 128;
-        case MS_LARGE:
-            return 192;
-        case MS_HUGE:
-            return 320;
-    }
-    return 0;
+    float ch = units::to_gram( weight ) * ( 0.40f - 2 * log( units::to_gram( weight ) ) );
+    const itype *chunk = item::find_type( get_meat_itype() );
+    return static_cast<int>( ch / units::to_gram( chunk->weight ) );
```

Comment:
`units::` is not needed when calling `to_gram` with a `units::mass` argument, the C++ compiler will find it anyway thanks to ADL.